src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/AbstractJavadocCheckTest.java

```diff
@@ -79,6 +79,15 @@ public void setUp(@SysErr Capturable systemErr) {
         systemErr.captureMuted();
     }
 
+    @Test
+    public void testSetJavadocTokens() throws Exception {
+        final DefaultConfiguration checkConfig = createModuleConfig(DummyCheck.class);
```

Comment:
please explain reason of Dummy usage ?
Creation of any Dummy and UT on method is very last resort.

is there such good is reason, lets explain it in method javadoc to let people see it is not normal.

Reply:
We are referring to these mutations https://github.com/checkstyle/checkstyle/blob/b22d623a919d198727e62e1dbb9855ebfc7af0bf/.ci/pitest.sh#L149-L151

```beginJavadocTree(root);```, ```finishJavadocTree(root);``` and ```javadocTokens.clear();``` are all unused in all our check so no existing Javadoc check needs them.

Reply:
JavadocMetadataScraper uses begin/finish please use it.

> `javadocTokens.clear();`

should be removed.
As we have similar method in java Checks
https://github.com/checkstyle/checkstyle/blob/24ead56cf961060af6e08399e999440ef6945f84/src/main/java/com/puppycrawl/tools/checkstyle/api/AbstractCheck.java#L95-L97
and it does not have 'clear' method, and it is expected behavior.

Here is good example how pitest found bug/inconsistency in behavior.

Reply:
> javadocTokens.clear(); should be removed.

done this this PR

> JavadocMetadataScraper uses begin/finish please use it.

I'm working on it. I'll send another PR.